mock/src/transaction.rs

```diff
@@ -171,7 +171,8 @@ impl Default for MockTransaction {
             block_hash: Hash::zero(),
             block_number: U64::zero(),
             transaction_index: U64::zero(),
-            from: AddrOrWallet::Addr(MOCK_ACCOUNTS[0]),
+            //from: AddrOrWallet::Addr(MOCK_ACCOUNTS[0]),
+            from: AddrOrWallet::random(&mut OsRng),
```

Comment:
why this change?

Reply:
if `from` is  AddrOrWallet::Addr, the signature inside witness::tx will be. {r:0, s:0, v:0}.   Switching to AddrOrWallet::Wallet will make the tx more "real".